ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController.swift

```diff
@@ -72,15 +63,14 @@ class NotificationViewDemoController: DemoController {
                 container.addArrangedSubview(UIView())
             }
             addTitle(text: variant.displayText)
-            container.addArrangedSubview(createNotificationView(forVariant: variant))
 
             let showButton = MSFButton(style: .secondary, size: .small, action: { [weak self] _ in
                 guard let strongSelf = self else {
                     return
                 }
 
-                strongSelf.createNotificationView(forVariant: variant).show(in: strongSelf.view) {
-                    $0.hide(after: variant.delayForHiding)
+                strongSelf.createNotificationView(forVariant: variant).showNotification(in: strongSelf.view) {
+                    $0.hide(after: 3.0)
```

Comment:
Just wondering out loud: do we maybe want to provide some constants for how long we expect a "normal" notification to be on screen? It'd help with consistency across clients, but could also just map to a standard `TimeInterval` so clients would still be free to put whatever they want in.

Reply:
This could definitely be a follow-up conversation after this PR, but wanted to jot my thoughts down while they were fresh :)

Reply:
I think I briefly discussed this with Rafael when we decided to remove the delayForHiding constant and we agreed to leave it fully up to implementer, but the consistency across clients is a good point. Will definitely take a second look at that! :)